include/devicetree/spi.h

```diff
@@ -116,6 +116,40 @@ extern "C" {
  */
 #define DT_SPI_DEV_HAS_CS_GPIOS(spi_dev) DT_SPI_HAS_CS_GPIOS(DT_BUS(spi_dev))
 
+/**
+ * @brief Get a SPI device's chip select GPIO controller's node identifier
+ *
+ * Example devicetree fragment:
+ *
+ *     gpio1: gpio@... { ... };
+ *
+ *     gpio2: gpio@... { ... };
+ *
+ *     spi@... {
+ *             compatible = "vnd,spi";
+ *             cs-gpios = <&gpio1 10 GPIO_ACTIVE_LOW>,
+ *                        <&gpio2 20 GPIO_ACTIVE_LOW>;
+ *
+ *             a: spi-dev-a@0 {
+ *                     reg = <0>;
+ *             };
+ *
+ *             b: spi-dev-b@1 {
+ *                     reg = <1>;
+ *             };
+ *     };
+ *
+ * Example usage:
+ *
+ *     DT_SPI_DEV_CS_GPIOS_CTLR(DT_NODELABEL(a)) // DT_NODELABEL(gpio1)
+ *     DT_SPI_DEV_CS_GPIOS_CTLR(DT_NODELABEL(b)) // DT_NODELABEL(gpio2)
```

Comment:
Shouldn't it be, rather:
```suggestion
 *     DT_NODELABEL(DT_SPI_DEV_CS_GPIOS_CTLR(a)) // DT_NODELABEL(gpio1)
 *     DT_NODELABEL(DT_SPI_DEV_CS_GPIOS_CTLR(b)) // DT_NODELABEL(gpio2)
```
?